rest_sensors.cpp

```diff
@@ -1179,6 +1179,9 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                         if (sendTuyaRequest(task, TaskThermostat, DP_TYPE_VALUE, 0x2c, data))
                         {
                             updated = true;
+                            
+                            rspItemState[QString("/sensor/%1/config/%2").arg(id).arg(rid.suffix)] = offset;
```

Comment:
The usage of `rid.suffix` is good but note that "config/" is already part of the suffix so it need to be left out when building the string:

`rspItemState[QString("/sensor/%1/%2").arg(id).arg(rid.suffix)] = offset;`

Reply:
done